Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -535,6 +536,7 @@ - (void)assertDefaultValues:(SentryOptions *)options
     XCTAssertNil(options.urlSessionDelegate);
     XCTAssertEqual(YES, options.enableSwizzling);
     XCTAssertEqual(NO, options.enableFileIOTracking);
+    XCTAssertEqual(YES, options.enableAutoBreadcrumbTracking);
```

Comment:
`m`: Please also add a test like `testEnableNetworkBreadcrumbs` for this new flag.
```
- (void)testEnableNetworkBreadcrumbs
{
    [self testBooleanField:@"enableNetworkBreadcrumbs"];
}
```